code-snippet-plugin/src/main/java/com/okta/maven/snippet/CodeSnippetMojo.java

```diff
@@ -115,15 +118,30 @@ public void execute() throws MojoExecutionException {
 
         final Map<String, String> methodBodyMap = new HashMap<>();
 
-
         new VoidVisitorAdapter<Object>() {
             @Override
             public void visit(final MethodDeclaration method, final Object arg) {
                 super.visit(method, arg);
 
                 String body = method.getBody().get().getStatements().stream()
-                        .map(Node::toString)
+                        .map(statement -> {
+                            TokenRange tokenRange = statement.getTokenRange().get();
+                            Optional<Comment> comment = statement.getComment();
+
+                            StringBuilder blockAString = new StringBuilder();
```

Comment:
Are you expecting to have multiple block strings in this method? If not, maybe we can change `blockAString` to `blockString` to make this more clear.

Reply:
ha, that was suppose to be `blockAsString` good catch :)

Reply:
Note the statement could be a single line or a multiple lines:

Single: `int foo = 1;`

or  multiple:

```java
 System
          .out.println("doing foo");
```

The second looks funny, but it is for testing purposes, a real example would look more like:
```java
SwaApplication swaApp = client.instantiate(SwaApplication.class)
    .setSettings(client.instantiate(SwaApplicationSettings.class)
    .setApp(client.instantiate(SwaApplicationSettingsApplication.class)
        .setButtonField("btn-login")
        .setPasswordField("txtbox-password")
        .setUsernameField("txtbox-username")
        .setUrl("https://example.com/login.html")));
```

This is a single statement.

It would be nicer to just get the raw text from the method block, but this was a relatively quick solution to the problem (given the API of the JavaParser)